### PR TITLE
Add new buildpack supporting chrome for tests in console

### PR DESCRIPTION
**Pull request by y-kkamil**
**Description**

Changes proposed in this pull request:

- Added missing chrome/puppeteer dependencies, so that console tests are able to run

### CODE REVIEW

File: prow/images/buildpack-node-chromium/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Node buildpack with chromium and its dependencies
+
+FROM eu.gcr.io/kyma-project/prow/bootstrap:0.0.1
```
Comment (michal-hudy) [inline prow/images/buildpack-node-chromium/Dockerfile]: It should extend Node Buildpack

Comment (y-kkamil) [reply prow/images/buildpack-node-chromium/Dockerfile]: fixed

File: prow/images/buildpack-node-chromium/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    fonts-liberation \
+    libappindicator1 \
+    libnss3 \
+    lsb-release \
+    xdg-utils \
+    wget \
+    chromium=70.0.3538.110-1~deb9u1 \
+    && apt-get clean
+
+RUN npm install -g eslint-config-react-app@^3.0.4 \
```
Comment (Demonsthere) [inline prow/images/buildpack-node-chromium/Dockerfile]: Will we require updates to those packages? If yes, I would insist on using a mechanism like [this](https://docs.npmjs.com/cli/shrinkwrap) so we can control the package dependencies and modify them outside of the Dockerfile

File: prow/images/buildpack-node-chromium/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    libxss1 \
+    libxtst6 \
+    ca-certificates \
+    fonts-liberation \
+    libappindicator1 \
+    libnss3 \
+    lsb-release \
+    xdg-utils \
+    wget \
+    chromium=70.0.3538.110-1~deb9u1 \
```
Comment (Demonsthere) [inline prow/images/buildpack-node-chromium/Dockerfile]: Please export the version string as an env or arg in the dockerfile

File: prow/images/buildpack-node-chromium/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,66 @@
+# Node buildpack with chromium and its dependencies
+
+FROM eu.gcr.io/kyma-project/prow/test-infra/buildpack-node:v20181119-afd3fbd
+
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
```
Comment (michal-hudy) [inline prow/images/buildpack-node-chromium/Dockerfile]: It is already done in node buildpack.

File: prow/images/buildpack-node-chromium/Dockerfile (hunk 2 above)
Comment (michal-hudy) [inline prow/images/buildpack-node-chromium/Dockerfile]: It is already done in node buildpack.

File: prow/images/buildpack-node-chromium/Dockerfile (hunk 3 above)
Comment (y-kkamil) [reply prow/images/buildpack-node-chromium/Dockerfile]: done.

File: prow/images/buildpack-node-chromium/Dockerfile (hunk 2 above)
Comment (y-kkamil) [reply prow/images/buildpack-node-chromium/Dockerfile]: true, fixed

Comment (y-kkamil) [reply prow/images/buildpack-node-chromium/Dockerfile]: n/a since it's removed from the dockerfile. should we raise an issue in the node-buildpack?

File: prow/images/buildpack-node-chromium/Dockerfile (hunk 4 above)
Comment (y-kkamil) [reply prow/images/buildpack-node-chromium/Dockerfile]: removed.

File: prow/images/buildpack-node-chromium/README.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+# Buildpack Node.js + Chromium Docker Image
```
Comment (bszwarc) [inline prow/images/buildpack-node-chromium/README.md]:
```suggestion
# Node.js buildpack with Chromium Docker image
```

I found in the docs that it is Node.js buildpack rather than Buildpack Node.jsm that's why the change :) For example here: https://docs.cloudfoundry.org/buildpacks/node/index.html

File: prow/images/buildpack-node-chromium/README.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+## Overview
+
+This folder contains the Buildpack with Chromium installed that is based on the [buildpack-node](../buildpack-node) image. Use it to run tests requiring Chromium.
```
Comment (bszwarc) [inline prow/images/buildpack-node-chromium/README.md]:
```suggestion
This folder contains the Node.js buildpack with Chromium installed. It is based on the [buildpack-node](../buildpack-node) image, and you can use it to run tests requiring Chromium.
```

File: prow/images/buildpack-node-chromium/Dockerfile [hunk 7]
```diff
@@ -0,0 +1,51 @@
+# Node buildpack with chromium and its dependencies
+
+FROM eu.gcr.io/kyma-project/prow/test-infra/buildpack-node:v20181119-afd3fbd
```
Comment (michal-hudy) [inline prow/images/buildpack-node-chromium/Dockerfile]: Change to same version as used in jobs: `eu.gcr.io/kyma-project/prow/test-infra/buildpack-node:v20181130-b28250b`

Comment (y-kkamil) [reply prow/images/buildpack-node-chromium/Dockerfile]: done
